library/src/test/java/org/zalando/failsafeactuator/endpoint/FailsafeEndpointTest.java

```diff
@@ -0,0 +1,75 @@
+package org.zalando.failsafeactuator.endpoint;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import net.jodah.failsafe.CircuitBreaker;
+import org.junit.Test;
+import org.zalando.failsafeactuator.service.CircuitBreakerRegistry;
+
+import static com.jayway.jsonpath.matchers.JsonPathMatchers.hasJsonPath;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+public class FailsafeEndpointTest {
+
+  private final ObjectMapper mapper = new ObjectMapper();
+
+  private final FailsafeEndpoint endpoint;
+  private final CircuitBreakerRegistry circuitBreakerRegistry;
+
+  public FailsafeEndpointTest() {
+    circuitBreakerRegistry = new CircuitBreakerRegistry();
+    endpoint = new FailsafeEndpoint(circuitBreakerRegistry);
+  }
+
+  @Test
+  public void singleBreakerClosed() throws JsonProcessingException {
+    CircuitBreaker test = circuitBreakerRegistry.getOrCreate("Test");
+    test.close();
+
+    String output = mapper.writeValueAsString(endpoint.invoke());
+
+    verifyBreaker(output, "Test", "CLOSED");
```

Comment:
Would be nice to introduce constants for the Strings.

Reply:
Okay. Added them.